src/content/docs/installation/installation_on_root.mdx

```diff
@@ -1,58 +1,225 @@
+
+10. Select the specific packages you wish to install from the provided list, and deselect any that you do not require.
+
+11. Setup your login credentials.
+
+12. Review the installation summary on the Overview Page carefully. If all the settings look correct for you, proceed with the installation by clicking on ***Install Now***. Otherwise, go back and make any necessary changes.
+
+</Steps>
+
+</TabItem>
+
+</Tabs>
+
+### MBR/BIOS
+
 :::note
-Secure boot must be disabled when installing CachyOS. To setup secure boot after install, see: [Secure Boot Setup](/configuration/secure_boot_setup)
+GRUB is the only boot manager available for a MBR/BIOS system.
 :::
 
-Calamares will then proceed to install CachyOS into your drive. The drivers will already be configured and working correctly out of the box with the help of our Hardware Detection. There is no need to install additional drivers after the installation unless needed for some less supported hardware.
+<Steps>
 
-As soon as Calamares has finished installing CachyOS, you'll get a prompt to reboot your system.
+1. Boot into the ISO and click the **Launch Installer** button
 
-### Common Issues
+2. Set your preferred **Language** and **Region/Timezone**.
 
-#### Key was created in the future
+3. Configure your **Keyboard Layout**.
 
-The Installation does fail with following input:
-```
+4. Select **Manual partitioning**.
+
+5. Create a new partition with the following:
+    * Size: ***At least 20000MiB***
+    * Filesystem: Refer to [Filesystem](/installation/filesystem)
+    * Mount point: ***/***
+    * Flags: 
+
+6. Double check that ***Install boot loader on:*** is pointing to your boot drive e.g: /dev/sda
+
+7. Pick the **Desktop Environment** you'd like to use, see [Desktop environments](/installation/desktop_environments).
+
+8. Select the specific packages you wish to install from the provided list, and deselect any that you do not require.
+
+9. Setup your login credentials.
+
+10. Review the installation summary on the Overview Page carefully. If all the settings look correct for you, proceed with the installation by clicking on ***Install Now***. Otherwise, go back and make any necessary changes.
+
+</Steps>
+
+## Common Issues
+
+### Key was created in the future
+
+:::note
+This will be fixed in the September 2024 Release.
+:::
+
+The Installation fails with the following error log:
+```text
 gpg: key B1D99E8932D24EB5 was created 4785 seconds in the future (time warp or clock problem)
 ```
 
-This indicates that the hardware clock is not correctly synced.
-After the installation failed reboot into the ISO and run following commands:
+This problem is caused by an out-of-sync local time.
+To be specific, the pacman master key was created when a wrong time set in your OS. This may cause the timestamp of the pacman master key is in the future after you correct the time on your OS (possibly by ntp).
```

Comment:
I don't how true this is

Reply:
is it that exact problem

Reply:
i brought this on because I've seen systems with synced times also come across this issue and using this command fixes it. I guess it might be bad memory.